autoprotocol/protocol.py

```diff
@@ -938,24 +940,33 @@ def liquid_handle_dispense(
 
         Parameters
         ----------
-        source : Well
-            Well(s) to transfer liquid from.
-        destination : Well or WellGroup or list(Well)
-            Well(s) to transfer liquid to.
+        source : Well or list(Well) or list(tuple(Well, int))
+            Well(s) to transfer liquid from. If the source is given as a list
+            of (Well, int) tuples (ie: source=[(Well, 2), (Well, 5)])
+            the Well is the source well and the int is the number of dispense
+            chips to use from the specified source.
+        destination : Well or WellGroup or list(Well) or list(WellGroup)
```

Comment:
in your example, this is a Container, not a wellgroup.

Reply:
Yeah, I need to update the example sorry